Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -540,6 +542,7 @@ open class ComposerVC: _ViewController,
     func bindMenuController() {
         menuController = ChatMenuViewController.instantiateController(storyboard: .wallet)
         menuController?.extraData = self.channelController?.channel?.extraData ?? [:]
+        menuController?.isChatOnly = self.channelController?.channel?.name == nil
```

Comment:
Is gift card flow only enable for 1-1 chat?
If yes, you can make condition with this method: `channelController?.channel?.isDirectMessageChannel`

Reply:
i have updated it. Thanks